src/data/sources/L1Profile.ts

```diff
@@ -10,7 +10,7 @@ export interface L1BufferJSON {
     'consumed-size-bytes': number;
     'percent-consumed': number;
     'reserved-size-bytes': number;
-    'start-address': `0x${number}`;
+    'start-address': number;
```

Comment:
Nitpick: let's keep a more specific type, the data is numbers in hex format.

Reply:
you are suggesting  `0x${0x123456}`. i find it excesive and uneeded